src/analysis/normed_space/basic.lean

```diff
@@ -789,6 +789,10 @@ def matrix.normed_space {α : Type*} [normed_field α] {n m : Type*} [fintype n]
   normed_space α (matrix n m α) :=
 pi.normed_space
 
+lemma matrix.norm_entry_le_entrywise_sup_norm {α : Type*} [normed_field α] {n m : Type*} [fintype n]
+  [fintype m] (M : (matrix n m α)) {i : n} {j : m}:
+  ∥M i j∥ ≤ ∥M∥ := @le_trans _ _ _ _ _ (norm_le_pi_norm (M i) j) (norm_le_pi_norm M i)
```

Comment:
```suggestion
  [fintype m] (M : (matrix n m α)) {i : n} {j : m} :
  ∥M i j∥ ≤ ∥M∥ :=
(norm_le_pi_norm (M i) j).trans (norm_le_pi_norm M i)
```

Reply:
The dot notation definitely reads cleaner here, thanks.